Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -791,6 +801,11 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var waypointStyle: WaypointStyle = .annotation
     
+    /**
+     Allows to control whether night style is used in dark mode or not.
+     */
+    public var usesNightStyleInDarkMode = true
```

Comment:
Let's set it to false to prevent unexpected changes in style for already existing users.